demisto_sdk/commands/download/downloader.py

```diff
@@ -160,271 +141,316 @@ def __init__(
         **kwargs,
     ):
         self.output_pack_path = output
-        self.input_files = [input] if isinstance(input, str) else input
+        self.input_files = input if isinstance(input, list) else [input]
```

Comment:
can we change `input` to a word that isn't a python builtin?

Reply:
This variable is received from `click` as kwargs because of the `--input` flag.
I don't _think_ there's a way without changing the flag.